CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Change Log
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and adheres to [Semantic Versioning](http://semver.org) as of version 1.0.0.
```

Comment:
Welp, for whatever such a commitment might mean, I'm in favor.  I don't want to invent our own system or assign our own semantics :grin: